datalad/resources/website/assets/js/main.js

```diff
@@ -205,8 +205,9 @@ function directory(jQuery, md5) {
     // add click handlers to each row(cell) once table initialised
     initComplete: function() {
       var api = this.api();
-      // all tables should have ../ parent path
-      if (!parent)
+      // all tables should have ../ parent path except webinterface root
+      var current_loc = absolute_url(getParameterByName('dir')).replace(/\/?$/, '') + '/';
```

Comment:
Yeah, for now this has been tested locally. 
Would you prefer the obscure_filename based test in this pr itself ?